src/settings.ts

```diff
@@ -235,18 +235,6 @@ export interface Template_v2 {
 
 // @debt Refactor this constant into types/templates or similar?
 export const BURN_VENUE_TEMPLATES: Array<Template> = [
-  {
-    template: VenueTemplate.conversationspace,
-    name: "Conversation Space",
-    description: ["A room of tables in which to talk and make merry."],
-  },
-  {
-    template: VenueTemplate.zoomroom, // keeping as zoom room for backward compatibility
-    name: "Experience",
-    description: [
-      "Ideal for performances, debates, interactive sessions of all kinds: a Zoom room with its own spot on the Jam",
-    ],
-  },
   {
     template: VenueTemplate.partymap,
     name: "Party Map",
```

Comment:
This and all other text occurrences of `party map` should be renamed to `camp`

Reply:
I was thinking that renaming is only related to the Admin form and fixed it here https://github.com/sparkletown/sparkle/pull/1930/files#diff-ef665bebb841378779cc61dd106ec01e676fb327a19ec2add06edaf05884acd0R348-R351

But your solution looks better! Thank you

Reply:
fixed in 345d697daa39b113824de2229eb71bbbe6d4bc28